save/README.md

```diff
@@ -0,0 +1,87 @@
+# Save action
+
+The save action saves cache just like the post step of the `cache` action, except it can be run individually now without having to call the restore step. The save action needs to be called with required inputs, the `key` with which the cache needs to be stored and the `path` that needs to be cached.
+
+## Inputs
+
+* `key` - 'An explicit key for saving the cache'
+* `path` - 'A list of files, directories, and wildcard patterns to cache'
+* `upload-chunk-size` - 'The chunk size used to split up large files during upload, in bytes'
+
+## Outputs
+
+This action has no outputs.
+
+## Use cases
+
+As this is a newly introduced action to give users more control in their workflows, below are some use cases where one can use this action.
+
+### Only save cache
+
+In case you are using separate jobs for generating common artifacts and sharing them across different jobs, this action will help you with your save only needs.
+
+```
+steps:
+  - uses: actions/checkout@v3
+
+  - name: Install Dependencies
+    if: steps.cache.outputs.cache-hit != 'true'
+    run: /install.sh
+
+  - name: Build common artifacts
+    run: /build.sh
+
+  - uses: actions/cache/save@v3
+    id: cache
+    with:
+      path: path/to/dependencies
+      key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+```
+
+### Re-evaluate cache key while saving
+
+Some technologies like dot-net generate the lockfiles during the build time, due to which the already evaluated `${{ hashFiles('**/lockfiles') }}` hash doesn't match the actual hash. Using save action with the same key will not re-evaluate the key as the hash was generated during restores. However passing the expression to re-evaluate the hash would generate a new key in the save action now as save would be called after the build step.
```

Comment:
Too verbose. Just mention that this action allows re-evaluation of key.

Reply:
Done